protobuf-converter/src/test/java/io/confluent/connect/protobuf/ProtobufConverterTest.java

```diff
@@ -521,6 +527,38 @@ public void testToConnectDataForValueWithNamespaceNested() throws Exception {
     assertEquals(expected, result);
   }
 
+  @Test
+  public void testToConnectDataForProtobufStruct() throws Exception {
+    Value v = Value.newBuilder().setNumberValue(123).build();
```

Comment:
nitpick: Is it possible to have the original schema written here as comment so that we know how the schema look like in text? Looking at the builder and the code not always obvious how the schema look like and it is time consuming.